blocks/featured/index.js

```diff
@@ -93,9 +23,95 @@ export default registerBlockType(
 			__( 'Travel' )
 		],
 
-		edit() {
-			return deamplify( renderStaticFeaturedBlock() );
+		attributes: {
+			heading: {
+				selector: '.travel-featured-heading',
+				source: 'children',
+				type: 'array'
+			}
 		},
+
+		edit: withAPIData( () => {
+			return {
+				featuredLocations: '/wp/v2/locations?per_page=6&meta_value=1&meta_key=amp_travel_featured'
+			};
+		} )( ( { featuredLocations } ) => { // eslint-disable-line
+			const hasLocations = Array.isArray( featuredLocations.data ) && 6 === featuredLocations.data.length;
+			if ( ! hasLocations ) {
+				return (
+					<Placeholder key="placeholder"
+								icon="admin-post"
+								label={ __( 'Locations' ) }
+					>
+						{ __( 'Not enough featured locations found, add at least six to use the block.' ) }
```

Comment:
Would it be easier to use if the message mentioned "Location" terms, maybe like:
```
Not enough featured locations found. Please add at least six "Locations" terms, select an image, and check "Featured destination."
```